flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/SortTest.scala

```diff
@@ -46,6 +46,134 @@ class SortTest extends TableTestBase {
     util.verifyExecPlan("SELECT * FROM MyTable ORDER BY a DESC, b")
   }
 
+  @Test
+  def testSortedResultIntoMultiSinks(): Unit = {
+    val env = util.tableEnv
+    env.executeSql(s"""
+                      |CREATE TABLE Source (
+                      |  a int,
+                      |  b bigint,
+                      |  c string,
+                      |  d string,
+                      |  e string
+                      | ) WITH (
+                      |  'connector' = 'values',
+                      |  'bounded' = 'true'
+                      |)
+                      |""".stripMargin)
+    env.executeSql("""
+                     |CREATE TABLE sink1 (
+                     |   a int,
+                     |   b bigint,
+                     |   c string
+                     |) WITH (
+                     |   'connector' = 'filesystem',
+                     |   'format' = 'testcsv',
+                     |   'path' = '/tmp/test'
+                     |)
+                     |""".stripMargin)
+    env.executeSql("""
+                     |CREATE TABLE sink2 (
+                     |   a int,
+                     |   b bigint,
+                     |   c string,
+                     |   d string
+                     |) WITH (
+                     |   'connector' = 'filesystem',
+                     |   'format' = 'testcsv',
+                     |   'path' = '/tmp/test'
+                     |)
+                     |""".stripMargin)
+    val stmtSet = env.createStatementSet()
+    stmtSet.addInsertSql("INSERT INTO sink1 select a, b, c from Source order by d")
+    stmtSet.addInsertSql("INSERT INTO sink2 select a, b, c, d from Source order by d")
+
+    util.verifyExecPlan(stmtSet)
+  }
+
+  @Test
+  def testSortedViewResultIntoMultiSinks(): Unit = {
```

Comment:
I think the test is more related to the {@link SubPlanReuseTest}. Could you move the test?

Reply:
> I think the test is more related to the {@link SubPlanReuseTest}. Could you move the test?

Done!